### ISSUE TITLE
3.0.3 broken on Node.js master

### ISSUE THREAD

**Issue by MylesBorins** [websockets/bufferutil#96]
Hey All,

it appears 3.0.3 is failing on Node.js master. This is due to an old version of nan... good news, y'all fixed it on master 🎉Would you be able to cut a patch release?

```
[bufferutil][(HEAD detached at f6fae26)]$ npm install --nodedir=~/code/node/master/

> bufferutil@3.0.3 install /Users/mborins/code/bufferutil
> prebuild-install || node-gyp rebuild

prebuild-install info begin Prebuild-install version 2.3.0
prebuild-install info install installing standalone, skipping download.
  CXX(target) Release/obj.target/bufferutil/src/bufferutil.o
In file included from ../src/bufferutil.cc:7:
In file included from ../node_modules/nan/nan.h:192:
../node_modules/nan/nan_maybe_43_inl.h:112:15: error: no member named 'ForceSet' in 'v8::Object'
  return obj->ForceSet(isolate->GetCurrentContext(), key, value, attribs);
         ~~~  ^

```

**Comment by lpinca** [comment]
Yes, will do later today.

**Comment by lpinca** [comment]
Released 3.0.4 with nan@2.10.0
